src/PortingAssistant.Client.Analysis/AnalysisHandler.cs

```diff
@@ -68,7 +68,7 @@ public class PortingAssistantAnalysisHandler : IPortingAssistantAnalysisHandler
                 if (analyzer == null || analyzer.ProjectResult == null)
                 {
                     _logger.LogError("Unable to build {0}.", project);
-                    return null;
+                    throw new PortingAssistantException($"Build project {project} failed", null);
```

Comment:
I think the return null is done on purpose. We want to filter out projects where analyzers are null so codelyzer can filter out invalid projects.

Reply:
Agree with you, I will close this PR. let me try if I can fix it through pure UI purpose.